plugins/chat/assets/javascripts/discourse/services/chat-audio-manager.js

```diff
@@ -59,10 +62,12 @@ export default class ChatAudioManager extends Service {
     }
 
     return audio.play().catch(() => {
-      // eslint-disable-next-line no-console
-      console.info(
-        "[chat] User needs to interact with DOM before we can play notification sounds."
-      );
+      if (!isTesting) {
```

Comment:
Why is `isTesting` not a function call here but it is further up?

Reply:
that's a mistake... I often do with isTesting... Thanks